agents/imap/general.py

```diff
@@ -3,78 +3,86 @@
 Description:
 General purpose functions to assist OWTF Agents
 '''
-import imp, os
+import imp
+import os
+import sys
 from collections import defaultdict
 
+
 class Plugin:
-	def Get(self, ModuleName, ModuleFile, ModulePath):# Python fiddling to load a module from a file, there is probably a better way...
-		f, Filename, desc = imp.find_module(ModuleFile.split('.')[0], [ModulePath]) #ModulePath = os.path.abspath(ModuleFile)
-		return imp.load_module(ModuleName, f, Filename, desc)
+    # Python fiddling to load a module from a file, there is probably a better way...
+    def Get(self, ModuleName, ModuleFile, ModulePath):
+        # ModulePath = os.path.abspath(ModuleFile)
```

Comment:
Dead/commented code should be removed as well, no? What's the point of using git otherwise